js/http/src/protocol.ts

```diff
@@ -1031,7 +1031,7 @@ export const wrapHttpProtocolBuilder = (_underlying: JvmHttpProtocolBuilder): Ht
   wsBaseUrls: (...urls) => wrapHttpProtocolBuilder(_underlying.wsBaseUrls(urls)),
   wsReconnect: () => wrapHttpProtocolBuilder(_underlying.wsReconnect()),
   wsMaxReconnects: (max) => wrapHttpProtocolBuilder(_underlying.wsMaxReconnects(max)),
-  wsAutoReplyTextFrame: (f) => wrapHttpProtocolBuilder(_underlying.wsAutoReplyTextFrame(f)),
+  wsAutoReplyTextFrame: (f) => wrapHttpProtocolBuilder(_underlying.wsAutoReplyTextFrame(f as Func<string, string>)),
```

Comment:
Ideally we'd want java2ts to generate the correct types based on NonNull/Nullable annotations... But that still wouldn't help much in this case (no way to know whether or not a `java.util.function.Function` should be expected to return `null`...).